features/gm.test.ts

```diff
@@ -75,3 +82,89 @@ Deno.test("group chat", async () => {
         pubkey: group_chat.groupKey.publicKey,
     }]);
 });
+
+Deno.test("should be only one group if the group created by me and invited me", async () => {
```

Comment:
English: There should only be one group if the group is created by me and inviting myself.